docker/web/development/php.ini

```diff
@@ -11,7 +11,7 @@ post_max_size = 192M
 
 max_input_vars = 15000
 
-error_reporting = -1
+error_reporting = E_ALL & ~E_NOTICE & ~E_STRICT & ~E_DEPRECATED & ~E_USER_DEPRECATED
```

Comment:
Please make sure that this does not end up in the final version of this PR, all deprecation warnings have been resolved by upgrading to ZF3/Laminas Framework with dependency injection.

The same applies to the changes to `.env.dist`, if you want to have it like that for everyone just change the default value. Otherwise you should only change it in `.env`.

Reply:
I did plan on updating `.env.dist` for everyone; that's now been done. Other changes have been reversed